README.md

```diff
@@ -1,114 +1,82 @@
 
-Each package handles its own release cycle as they are fundamentally independent and should not be locked to be up to date unless we want to.
+# How It All Fits Together
 
-Check each package's README for more information on how each one releases.
+### Vincent Developer Workflow
 
-### Summary
+1. Registers a Vincent App using the [Vincent App Dashboard](https://dashboard.heyvincent.ai/), selecting the from the existing Tools and Policies their App will use
+2. Optionally creates and publishes custom Tools and Policies tailored to their App’s functionality
+3. Integrates the Vincent App SDK to authenticate users and redirect them to the Vincent Consent Page for delegation
+4. Executes Tools on behalf of authenticated users, governed by the Vincent Policies each user has configured for their App
 
-- All apps hosted in vercel (dashboard) are built and made public with their own url. Then promotion to production is done manually or when merging to `main`.
-- SDK is released by `nx` with `pnpm release` (run at root)
+### Vincent User Workflow
 
-# Ecosystem
+1. Visits a Vincent App and is redirected to the Vincent Consent Page to log in or create their Agent Wallet
+2. Reviews the App’s requested Tools, configures Policy parameters, and approves the delegation
+3. Is redirected back to the App with a signed Vincent JWT, enabling the App to execute the approved Tools on their behalf within the user’s configured Policy limits
 
-## Policies
+# Why Vincent?
 
-- Spending Limits Contract
-  https://github.com/LIT-Protocol/SpendingLimitsContract
+Vincent optimizes for security, interoperability, and user-control to redefine how agents interact across Web3 and beyond:
 
-# Contributing
+- **Decentralized Key Management**: Vincent leverages Lit Protocol's [Programmable Key Pairs](https://developer.litprotocol.com/user-wallets/pkps/overview) (PKPs) to securely manage agent identities without exposing private keys.
+- **Verifiable, Fine-Grained Policies**: Users have full control over the policies they set and can revoke them at any time. Policies are fine-grained to specific operations and verifiable on-chain.
+- **Cross-Platform Automation**: With Vincent, agents can operate seamlessly across any blockchain or off-chain platform. Build agent-powered apps and tools for DeFi, TradFi, E-Commerce, and more.
+- **Agent Marketplace**: Vincent will eventually serve as a marketplace for Vincent-powered apps and tools, enabling them to be discovered and interacted with by end users.
+- **Open Source**: Fork, customize, and contribute to the [Vincent codebase](https://github.com/LIT-Protocol/Vincent) to shape the future of the agent-driven and user-controlled Web.
+
+# Useful Links
 
-Check [CONTRIBUTING.md](./CONTRIBUTING.md) for setup and general development instructions.
+- [Vincent Docs](https://vincent-docs-3rxwrggg2-lit-protocol.vercel.app/modules/Vincent_Docs.html)
+- [Automated Dollar-Cost Averaging Demo](https://demo.heyvincent.ai/)
```

Comment:
Link its repo too. We want devs to be able to fork that and create apps based on that structure